auras.py

```diff
@@ -43,12 +43,10 @@ def analyze_curses(char_stats, active_skills) -> list[list[str]]:
 
 	@staticmethod
 	def ascendancy_mod(aura_counter: list[int], char_stats: stats.Stats, node_name: str) -> list[str]:
-		# TODO: champion
 		if node_name == 'Champion':
 			return [
 				'// Champion',
 				'Enemies Taunted by you take 10% increased Damage',
-				'Your Hits permanently Intimidate Enemies that are on Full Life',
```

Comment:
wait why'd you get rid of this?

Reply:
The way this mod works is that it permanently sets the "Enemy is intimidated" flag in PoB with no way of removing it manually via the checkboxes, because it thinks that it's your (i.e. the carry's) mod.
In reality this will almost never be active in most cases since it relies on the aurabot to land the first hit.

My prefered solution would be to have a few condition checkboxes available for the website user to check, like "Is Vaal Haste active?", "Did your partymember land the first hit on the monster?", "Did your party member attack recently?" like the PoB config tab.
But that would require some messing with the frontend and I'm really bad with that.

Reply:
ohhh I realized that's what it was doing but forgot the "on full life" part. ok!